plugins/woocommerce/includes/abstracts/abstract-wc-settings-api.php

```diff
@@ -944,26 +944,14 @@ public function validate_password_field( $key, $value ) {
 	/**
 	 * Validate Textarea Field.
 	 *
+	 * @since 9.0.0 No longer allows storing IFRAME, which was allowed for "ShareThis" integration no longer found in core.
```

Comment:
one final question: would it still be valuable to allow IFRAMEs _if_ they are for ShareThis? we could use the HTML API to strip away tags except for those IFRAMEs pointing to a ShareThis URL…

should not stand in the way of this change, but it could be a way to avoid some potential backwards-compatability breakage.

Reply:
I don't think we should set that precedent of allowing some iframes but not others. This would also need handling at render time which could introduce some other issues. The ShareThis integration has "N/A" users and was permanently closed in 2020 so I think its ok to ignore that extension with this change.

The main concern is some other extension we're not aware of taking advantage of this, but I don't think we do much there beyond the advisory.